request-handler/lambda_function.py

```diff
@@ -1,12 +1,13 @@
-# Code added by Mike but written by Karen ;)
+# Added by mike; written by karen
+# View latest lambda function here: https://console.aws.amazon.com/lambda/home?region=us-east-1#/functions/ce-rap-karen-request-handler/versions/$LATEST?tab=graph
+
 import re
 import xml.etree.ElementTree as ET
 from urllib.request import urlopen
 
 
-def get_listing(prefix, suffix, bucket='ce-baked-ce-distribution-373045849756'):
-    bucket_name = 'ce-rap-karen'
-    url = f'https://{bucket_name}.s3.amazonaws.com/?list-type=2&prefix={prefix}'
+def get_listing(prefix, suffix, bucket='ce-rap-karen'):
```

Comment:
You need to change the bucket name again :D

Reply:
ooops.. moving to quickly. ⚡️ fixed!